tests/templates/test_layers/test_gate_fabric.py

```diff
@@ -0,0 +1,778 @@
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.70710678 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.70710678 + 0.0j,
+                        0.0 + 0.0j,
+                    ]
+                ),
+            ),
+            (
+                qml.math.array([0, 1, 1, 1]),
+                qml.math.array(
+                    [
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.70710678 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.70710678 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                    ]
+                ),
+            ),
+            (
+                qml.math.array([1, 1, 0, 1]),
+                qml.math.array(
+                    [
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        -0.70710678 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.70710678 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                    ]
+                ),
+            ),
+            (
+                qml.math.array([1, 1, 1, 1]),
+                qml.math.array(
+                    [
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        0.0 + 0.0j,
+                        1.0 + 0.0j,
+                    ]
+                ),
+            ),
+        ],
+    )
+    def test_decomposition_q(self, init_state, exp_state, tol):
+        """Test the decomposition of the Q_{theta, phi}` exchange gate by asserting the prepared
```

Comment:
Should be changed to :math:`Q_{theta, phi}`. Is it just a general decomposition test? (what is Q exchange gate?)

Reply:
Yeah, fixed it. This is just to test correctness of `Q` gate (removed the word `exchange`) which makes up the GateFabric template.